app/assets/stylesheets/alchemy/image_library.scss

```diff
@@ -0,0 +1,137 @@
+.picture-detail-bottom {
+  @include box-sizing(border-box);
+  position: absolute;
```

Comment:
Properties should be ordered background-color, bottom, height, left, padding-top, position, width